Precipitation Trigger

jeredb opened the pull request:
Precipitation trigger is similar to the temperature trigger:

`<Activate>
Precip: >0
</Activate>
<Deactivate>
Precip: =0
</Deactivate>`

Commit:
Precipitation Trigger

Precipitation trigger is similar to the temperature trigger:

`<Activate>
Precip: >0
</Activate>
<Deactivate>
Precip: =0
</Deactivate>`

joebuhlig commented:
Thanks for this, @jeredb. I love the idea of having precipitation trigger a project.

Help me out, though. I see you're storing the precipitation value in "condition", but I don't see that condition being passed into any of the extracted functions. I didn't run any tests on this, but it doesn't look like it would work. 

Am I missing something?

Commit:
Fixed condition call

`condition` wasn’t being called by the appropriate functions, but
should be now.

jeredb commented:
You are right, I managed to leave it out in a couple of places. I think I fixed it.

Documentation should include the sensitivity. The data returned can be 0.18, which would not trigger if the `<Activate>` tag is set to `Precip: =0`.
